introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>No need to put it in bold.</p><h3>Please add: , and specify id of any model available in HugginFace <a href="https://huggingface.co/models?pipeline_tag=fill-mask&amp;sort=downloads" target="_blank">Fill-Mask</a> and <a href="https://huggingface.co/models?pipeline_tag=text-classification&amp;sort=downloads" target="_blank">Text-Classification</a>, in the HF_MODEL_ID variable below.</h3><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>